src/api.test.ts

```diff
@@ -1,22 +1,149 @@
-import app from "./app"
-import request from "supertest"
+import app from "./app";
+import request from "supertest";
+import nock from "nock";
+
+import randomSuccessResponse_01 from "./testing_assets/json/random_success_01.json";
+import randomSuccessResponse_02 from "./testing_assets/json/random_success_02.json";
```

Comment:
seems like we can mention whether this response is for "query" or just random.
ex) 
`random_success_01`
`random_query_success_01`

whatever